appveyor.yml

```diff
@@ -1,18 +1,42 @@
-version: 1.0.{build}
-configuration: Debug
 platform: Any CPU
 image: Visual Studio 2017
 install:
 - git submodule update --init --recursive
-# force update ILSpy-tests to the latest ref
-- git submodule update --remote ILSpy-tests
+- ps: |
+    function gitVersion(){
```

Comment:
:bulb: It would be good to move this to a separate file, e.g. **BuildTools/appveyor-install.ps1**. It makes the script much easier to edit.